powersimdata/scenario/create.py

```diff
@@ -149,6 +149,7 @@ def get_bus_demand(self):
 
         :return: (*pandas.DataFrame*) -- data frame of demand (hour, bus).
         """
+        self._update_scenario_info()
```

Comment:
It makes sense. I am confused to why it was working before without the update.

Reply:
`get_demand` uses `get_profile` which uses `self.builder.demand`, but `get_bus_demand` passes `self.scenario_info` without updating `self.scenario_info` with what we're setting in the builder.

Reply:
Did we test it in Create state after #388?

Reply:
Probably not but I thought this was a feature you were using before creating each scenario. That's why I am surprised you just found the bug now.

Reply:
Nope, this method was not part of my usual scenario setup process. REISE.jl does the same thing within the Julia code though. This feature was mostly being used for scenarios in Analyze state.